perun-core/src/test/java/cz/metacentrum/perun/core/entry/ConsentsManagerEntryIntegrationTest.java

```diff
@@ -468,6 +418,259 @@ public void changeConsentStatusToSameValue() throws Exception {
 		assertThatExceptionOfType(InvalidConsentStatusException.class).isThrownBy(() -> perun.getConsentsManagerBl().changeConsentStatus(sess, consent, status));
 	}
 
+	@Test
+	public void evaluateConsentsIsTurnedOffOnInstance() throws Exception {
+		System.out.println(CLASS_NAME + "evaluateConsentsIsTurnedOffOnInstance");
+
+		Service service = setUpService();
+		Vo vo = setUpVo();
+		Member member = perun.getMembersManager().createMember(sess, vo, user);
+		Resource resource = setUpResource(facility, vo);
+		perun.getResourcesManagerBl().assignService(sess, resource, service);
+		AttributeDefinition attrDef = setUpUserAttributeDefinition("testUserAttribute");
+		perun.getServicesManagerBl().addRequiredAttribute(sess, service, attrDef);
```

Comment:
Yes, I thought about that and decided to extract there only facility and user because they are used in almost every test. However, I changed my mind and moved other objects there as well since it saves a lot of space and shouldn't have performance problems (creating unnecessary objects, but not in so many tests).